### PR TITLE
Test

### PR DESCRIPTION
**Pull request by JoshBowdenConcepts**
Opening this PR to show you some ways you could start to write your code in a more modern way. I will include how types are usually created in the industry. When you make more components use these updates as your examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,5 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
 Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+
+Test
```
Comment (dustinsoos) [inline README.md]: Do we need this here?

File: app/components/button.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import React from "react";
-import { ReactNode } from "react";
+import React, { ReactNode }  from "react";
 
-const Button = (props: { children: ReactNode }) => {
-  return <button>{props.children}</button>;
+type ButtonProps = React.HTMLAttributes<HTMLButtonElement>
```
Comment (dustinsoos) [inline app/components/button.tsx]: I would like to add a property here for size, how would I do that?

File: README.md (hunk 1 above)
Comment (JoshBowdenConcepts) [reply README.md]: I agree we can remove this. I have updated it in this SHA https://github.com/dustinsoos/bored-user/pull/1/commits/c37d56026dce1e1230588e6b741861f6e9f9880f

File: app/components/button.tsx (hunk 2 above)
Comment (JoshBowdenConcepts) [reply app/components/button.tsx]: I have updated this and added the property. I won't add it to the props for the component until we decide how it is to be used. https://github.com/dustinsoos/bored-user/pull/1/commits/d2d45613073e885bccd42e25bbc5ba8db906a36d

**Review by dustinsoos — APPROVED** [review]

**Pull request merged**